compiler/src/verify.rs

```diff
@@ -1,44 +1,12 @@
+use number::FieldElement;
 use std::{fs, path::Path, process::Command};
 
-use mktemp::Temp;
-use number::FieldElement;
+use crate::compile_asm_string;
 
 pub fn verify_asm_string<T: FieldElement>(file_name: &str, contents: &str, inputs: Vec<T>) {
-    let (pil_file_name, temp_dir) = compile_asm_string_temp(file_name, contents, inputs);
-    verify(&pil_file_name, &temp_dir);
-}
-
-#[allow(unused)]
-pub fn compile_asm_string_temp<T: FieldElement>(
-    file_name: &str,
-    contents: &str,
-    inputs: Vec<T>,
-) -> (String, Temp) {
-    let pil = pilgen::compile(Some(file_name), contents).unwrap();
-    let pil_file_name = "asm.pil";
     let temp_dir = mktemp::Temp::new_dir().unwrap();
-    assert!(crate::compile_pil_ast(
-        &pil,
-        pil_file_name.as_ref(),
-        &temp_dir,
-        Some(|query: &str| {
-            let items = query.split(',').map(|s| s.trim()).collect::<Vec<_>>();
-            assert_eq!(items.len(), 2);
-            match items[0] {
-                "\"input\"" => {
```

Comment:
Haha interesting. The problem was that this does not implement "print", but the one used in compile_asm_string does :)